src/sklearn_evaluation/plot/target_analysis.py

```diff
@@ -0,0 +1,154 @@
+"""
+Plot for target analysis
+
+NOTE: this is based on the yellowbricks target analysis module. License below.
+
+Copyright 2016-2020 The scikit-yb developers
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.utils.multiclass import unique_labels, type_of_target
+
+
+def _validate_target(y):
+    """
+        Raises a value error if the target is not a classification target.
+        """
+    # Ignore None values
+    if y is None:
+        return
+
+    y_type = type_of_target(y)
+    if y_type not in ("binary", "multiclass"):
+        raise TypeError(
+            "'{}' target type not supported, only binary and multiclass".
+            format(y_type))
+
+
+def target_plot(y_train, y_test=None, labels=None, colors=None, ax=None):
```

Comment:
We need to track the usage of this method. We can do it with `@SKLearnEvaluationLogger.log(feature='plot')`